File: 0791A.Bear.and.Big.Brother/solution.py
import math


def solution():
    limak, bob = [float(x) for x in input().split()]

    result = (math.log2(bob) - math.log2(limak)) / (math.log2(3) - math.log2(2))
    if abs(result - int(result)) < 1e-8:
        result += 1
    else:
        result = int(result) + 1

    print(int(result))


if __name__ == "__main__":
    solution()

File: 0791A.Bear.and.Big.Brother/solution.go
package main

import (
	"bufio"
	"fmt"
	"math"
	"os"
)

var reader *bufio.Reader = bufio.NewReader(os.Stdin)
var writer *bufio.Writer = bufio.NewWriter(os.Stdout)

func Fprintf(format string, x ...interface{}) {
	fmt.Fprintf(writer, format, x...)
}

func Fscanf(format string, x ...interface{}) {
	fmt.Fscanf(reader, format, x...)
}

func Solution() {
	var limak, bob float64
	Fscanf("%f %f\n", &limak, &bob)

	result := (math.Log(bob) - math.Log(limak)) / (math.Log(3) - math.Log(2))
	if result == float64(int(result)) {
		result++
	} else {
		result = float64(int(result) + 1)
	}

	Fprintf("%d\n", int(result))
}

func main() {
	Solution()
	writer.Flush()
}
